### PR TITLE
Nextjs paths

### PR DESCRIPTION
**Pull request by tylergeorges**
Added semantic path linking in TSConfig Paths

### CODE REVIEW

File: src/components/Core/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { styled, css, theme } from '@/../stitches.config';
+import { styled, css, theme } from '@stitches-config';
```
Comment (Yomanz) [inline src/components/Core/index.tsx:1]: Is there any reason this cant just be @stitches?

**Review by Yomanz — CHANGES_REQUESTED** [review]

File: src/pages/channels/[guild]/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
+import { getEnvVar } from '@util/env';
```
Comment (mason-rogers) [inline src/pages/channels/[guild]/index.tsx:1]: this can be changed to `from '@util'`

**Review by mason-rogers — CHANGES_REQUESTED** [review]
looks good other than that single thing, and agree with @Yomanz that @stitches-config should probably just be called @stitches for cleanliness

File: src/pages/channels/[guild]/index.tsx (hunk 2 above)
Comment (tylergeorges) [reply src/pages/channels/[guild]/index.tsx:1]: alright changing them now

**Review by Yomanz — APPROVED** [review]
LGTM

**Pull request merged**